src/components/format/DurationTime.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import formatTimeInterval from '../../utils/format-time-interval';
+
+export default class DurationTime extends React.PureComponent {
+	static propTypes = {
+		className: PropTypes.string,
+		children: PropTypes.node
+	};
+
+	static defaultProps = {
+		className: '',
+		children: null
+	};
+
```

Comment:
add static methods `format()` and `unformat()` (leave it empty). Use utility funcitons in `format()` and if necessary convert types